PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Pull Request Template
+
+---
+## Protocol Information
+
+- **Name:**
+- **Twitter Link:**
+- **Website Link:**
+- **Github:**
+- **Developer Docs:**
+- **Are you a member of the protocol?** [Yes/No]
+
+## Checklist:
+
+- [ ] Added export `./{protocolId}` in `src/logics/index.ts`
+- [ ] Exported all Logics in `src/logics/{protocolId}/index.ts`
+- [ ] Ensured that all Logics have unit tests
```

Comment:
```suggestion
- [ ] Ensured that all Logics have passed unit tests
```

Reply:
This check is intended to emphasize the importance of writing tests. It's not about the person submitting the PR checking whether all unit tests are passing.